00-spatio-temporal-science-questions.Rmd

```diff
@@ -225,214 +187,209 @@ achieve this goal we need data that:
+	Source: National Ecological Observatory Network (NEON) figure based on Landsat
+	Data.
+	</figcaption>
+</figure>
+
+* <a href="http://earthobservatory.nasa.gov/Features/MeasuringVegetation/measuring_vegetation_2.php" target="_blank"> More on NDVI from NASA</a>
 
-<i class="fa fa-star"></i> **Data Note:** NDVI can be calculated using any image
+<i class="fa fa-star"></i> **Data Tip:** NDVI can be calculated using any image
 that contains a near-infrared band. Some phenocams capture near-infrared light.
 {: .notice}
 
 ## Find Remote Sensing Data 
 
-There are many sources that allow you to request and freely download remote sensing 
-imagery and products like NDVI. In the United States, one popular data provider is the United States 
-Geological Survey.
+There are many sources that allow us to request and freely download remote
+sensing imagery and products like NDVI. In the United States, one popular data
+provider is the United States Geological Survey.
 
 * <a href="http://phenology.cr.usgs.gov/get_data_main.php" target="_blank">Remote Sensing Phenology</a>
 * <a href="http://glovis.usgs.gov/" target="_blank">USGS GLOVIS</a>
 * <a href="http://landsatlook.usgs.gov/" target="_blank">Landsat Look Viewer</a>
-* <a href="http://landsat.gsfc.nasa.gov/" target="_blank">NASA Landsat</a>
 
-## Landsat Look Viewer
+Other, non-USGS sources of remote sensing data can be found in the 
+[Data Sources list at the end of this tutorial]({{ site.baseurl}}/R/spatio-temporal-research-questions#resources-for-locating-spatial-and-ecological-data).
 
 <div id="challenge" markdown="1">
+## Challenge: Explore *Landsat Look* Website
 
-## Challenge: Explore Landsat Look Website
-
-Visit the <a href="http://landsatlook.usgs.gov/" target="_blank">Landsat Look Viewer
-website</a>. Search for an location - for example `Harvard Forest, Massachusetts, 
-United States`. 
+Visit the 
+<a href="http://landsatlook.usgs.gov/" target="_blank">Landsat Look Viewer website</a>.
+Search for any location of interest. Can decide? Try one of our study sites: 
+Harvard Forest, Massachusetts, United States. 
 
+1. Data from which Landsat sensors are available for your area? 
+2. How is the quality of the images?
 </div>
 
 ```{r explore-landsat-data, echo=FALSE}
-#.
+# Answers for Harvard Forest; will vary by location
+# 1. Landsat 8 (good quality), Landsat 7 EMT+ SLC-off (some problem areas), 
+# Landsat 7 EMT+ SLC-on (one strip of missing data), Landsat 4-5 (good quality),
+# Landsat 1-5 (missing large amounts of data). 
 
 ```
 
-### Raster Data Format - Work With Remote Sensing Imagery 
-
-Remote sensing images are in **raster** format. They are composed of pixels - just like
-the pixels in the images we take with our camera phones. To work with raster data 
-in R, we can use the <a href="https://cran.r-project.org/web/packages/raster/index.html" target="_blank">`raster` package. </a>
-
-Because we are studying phenophase timing, we need raster data collected 
```

Comment:
These sections (and figure) were removed because we are pulling too much information into the intro lesson/overview. 

This is the first lesson of the intro and we haven't introduced the concepts of Raster and Vector (this is lesson 02) yet here we mention that RasterVis is a good package to view it in.  I've used the "Work with XX in R" sections to link out to the different series throughout this intro.  Since the raster time series is included within raster data series I wasn't going to link to both, however, I can add that link back in.